Extensions/Taxonomies/ITaxonomySetupHandler.cs

```diff
@@ -21,5 +21,13 @@ public interface ITaxonomySetupHandler : IEventHandler
         /// <param name="taxonomyName">Names of the Taxonomies to create.</param>
         /// <param name="termNames">Names of the Terms to create and add to the Taxonomy.</param>
         void ApplyTaxonomyCreationWithTerms(string taxonomyName, List<string> termNames);
+
+        /// <summary>
+        /// Applies the previously invoked creation of Taxonomy with Terms and Taxonomy settings.
+        /// </summary>
+        /// <param name="taxonomyName">Names of the Taxonomies to create.</param>
+        /// <param name="termNames">Names of the Terms to create and add to the Taxonomy.</param>
+        /// <param name="settings">The settings to add to the Taxonomy.</param>

```

Comment:
Ezeket a settingeket nem a Taxonomy-nak állítjuk be, hanem az általa létrehozott Term Type-nak (tehát pl. a Language Taxonomy-hoz automatikusan létrejön egy "Language Term" content type, aminek content itemjei a termek).